Update to conditional statement

RosCraddock opened the pull request.

RosCraddock committed:
Update to conditional statement

RosCraddock committed:
Minor Correction

Minor correction to allow the input of the alternative allele frequency by the user. This is now working!

gregorgorjanc commented on InputOutput.py:
```diff
@@ -236,7 +236,8 @@ def readInPedigreeFromInputs(pedigree, args, genotypes=True, haps=False, reads=F
     
     aapfile = getattr(args, "alt_allele_prob_file", None)
     if aapfile is not None:
-        pedigree.readInAAP(aapfile)
```
@RosCraddock I am not versed/familiar with `pedigree.readInAAP()`. Do we pass in a file or `aap`?

RosCraddock replied:
@gregorgorjanc This is a new function. Here, we pass in a file path through aap. This file contains the user-input alternative allele frequencies for each metafounder.

RosCraddock committed:
Error Message

Implementation of an error message if the user inputs a main_metafounder that does not start with "MF_".

gregorgorjanc reviewed (approved):
LGTM
